docs-js/faq.mdx

```diff
@@ -0,0 +1,27 @@
+---
+id: frequently-asked-questions
+title: Frequently Asked Questions
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: FAQ
+description: You asked! We answered! We collected here the most frequent question about SAP Cloud SDK for JavaScript.
+keywords:
+  - sap
+  - cloud
+  - sdk
+  - ai
+---
+
+### Which library should I use for AI integration in the SAP ecosystem? I see an npm package `CAP LLM Plugin`.
+
+Use the **SAP Cloud SDK for AI**, the only official Software Development Kit (SDK) for **SAP AI Core**, **SAP Generative AI Hub**, and **Orchestration Service**.
```

Comment:
I would phrase it a little less confrontative, therefore remove the "only" and maybe phrase it more like:

"Use the **SAP Cloud SDK for AI**, as it is the official Software Development ..."